recipe/meta.yaml

```diff
@@ -1,39 +1,48 @@
 {% set name = "pivy" %}
-{% set version = "0.6.8" %}
+{% set version = "0.6.9" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   fn: {{ name }}.tar.gz
-  url: https://github.com/FreeCAD/pivy/archive/{{ version }}.tar.gz
-  sha256: c443dd7dd724b0bfa06427478b9d24d31e0c3b5138ac5741a2917a443b28f346
+  url: https://github.com/coin3d/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c207f5ed73089b2281356da4a504c38faaab90900b95639c80772d9d25ba0bbc
 
 build:
-  number: 2
+  number: 0
+  script_env:
+    - USE_QT6=1  # [py >= 312]
+    - USE_QT6=0  # [py < 312]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ stdlib("c") }}
     - {{ compiler("cxx") }}
     - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - msinttypes      # [win]
     - cmake
     - ninja
     - swig
   host:
     - python
     - coin3d
-    - soqt
-    - qt-main
+    - soqt      # [py < 312]
+    - qt-main   # [py < 312]
+    - soqt6     # [py >= 312]
+    - qt6-main  # [py >= 312]
```

Comment:
Same question as before: is it really the intention to switch to qt6 only for py>=312? This is going to lead to weird situations in other feedstocks and the solver.

Is there a technical reason why python 3.9-3.11 cannot similarly switch?

Reply:
python 3.12 is failing with qt5 as pyside2 is not available for python3.12. But you are right, let's build the full build-matrix (qt5/qt6 x python3.9-python3.12)